### PR TITLE
Rename @inrupt to @solid

### PR DESCRIPTION
**Pull request by ajacksified**
After this merge, the github project should be renamed to `lit-term-js`.

### CODE REVIEW

**Comment by NSeydoux** [comment]
I thought the current proposal for this package was `@solid/lit-term`, `vocab` being used as a scope for generated vocabularies. Is this in scope for this PR ?

**Comment by ajacksified** [comment]
Thanks for the catch - updated to `lit-term`.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 on: 
   release: 
     types: [published]
+
 jobs:
   release-vocab-term:
-    runs-on: ubuntu-latest
-    name: Releases the lit-vocab-term package
+    runs-on: ubuntu-18.04
```
Comment (pmcb55) [inline .github/workflows/release.yml:7]: Can we use 20.04 instead...?

Comment (NSeydoux) [reply .github/workflows/release.yml]: Fixed in a16c9aeecef4fdbe52ef41ee7cd840572e46a0db

**Pull request merged**